### PR TITLE
[charts/karavi-metrics-powerflex] [charts/karavi-topology] Fix nginx configuration to work for Openshift deployment

### PR DESCRIPTION
**Pull request by ameer-jabbar**
#### Is this a new chart?

No.

#### Which issue(s) is this PR associated with:

- #61 

#### Checklist:

[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]

- [x] Chart Version bumped
- [ ] Variables are documented in the chart README.md
- [x] Title of the PR starts with the chart name (e.g. `[charts_dir/mychartname]`) if applicable

[e2e_test_ocp_4.5.log](https://github.com/dell/helm-charts/files/5662732/e2e_test_ocp_4.5.log)
[e2e_test_ocp_4.6.log](https://github.com/dell/helm-charts/files/5662734/e2e_test_ocp_4.6.log)

### CODE REVIEW

File: charts/karavi-metrics-powerflex/Chart.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: v2
-appVersion: "0.1.0"
+appVersion: "0.1.1"
```
Comment (tdawe) [inline charts/karavi-metrics-powerflex/Chart.yaml:2]: Should this be at version 0.2.0 so it matches the upcoming 0.2.0 release of the karavi-topology image release?

File: charts/karavi-metrics-powerflex/Chart.yaml [hunk 2]
```diff
@@ -1,5 +1,5 @@
...
 description: The open-source solution that provides Kubernetes administrators insight into storage usage and performance for containerized applications using Dell products.
 name: karavi-metrics-powerflex
-version: 0.1.0
+version: 0.1.1
```
Comment (tdawe) [inline charts/karavi-metrics-powerflex/Chart.yaml:5]: Should this helm chart version also be bumped up to 0.2.0, and not a patch version?

File: charts/karavi-metrics-powerflex/Chart.yaml (hunk 1 above)
Comment (ameer-jabbar) [reply charts/karavi-metrics-powerflex/Chart.yaml:2]: Yep - have to change that.

File: charts/karavi-metrics-powerflex/nginx.conf [hunk 3]
```diff
@@ -2,13 +2,16 @@ worker_processes  1;
...
+pid /tmp/nginx.pid;
+
 http {
     include       mime.types;
     default_type  application/octet-stream;
     sendfile        on;
     keepalive_timeout  65;
     server {
-      listen       443 ssl;
+      listen       8443 ssl;
```
Comment (tdawe) [inline charts/karavi-metrics-powerflex/nginx.conf:14]: Can you also update https://github.com/dell/karavi/blob/main/docs/karavi-observability/README.md in the `karavi` repository to reference port 8443 for the otel-collector.

**Comment by tdawe** [comment]
Should this go into the release branch instead of main?

File: charts/karavi-metrics-powerflex/nginx.conf (hunk 3 above)
Comment (ameer-jabbar) [reply charts/karavi-metrics-powerflex/nginx.conf:14]: Thanks, updated.

**Pull request merged**